objects/src/block/header.rs

```diff
@@ -275,6 +275,39 @@ impl Deserializable for BlockHeader {
     }
 }
 
+
+
+/// BLOCK NUMBER STRUCT
+
+/// Holds `u32` type  to signify Block Number
+
+
+#[derive(Debug, Eq, PartialEq, Copy, Clone, PartialOrd, Ord)]
```

Comment:
I think we should also derive `Hash` so this can be used in a `HashMap`. We should also implement `Display` so we can use `BlockNumber` in error messages.